contracts/LightMigrator/LightMigrator.sol

```diff
@@ -34,10 +35,25 @@ contract LightMigrator is ILockDealV2, FirewallConsumer {
      * @param _nft Address of the LockDealNFT contract.
      * @param _oldVault Address of the old DelayVault (IDelayVaultV1) contract.
      */
-    constructor(ILockDealNFT _nft, IDelayVaultV1 _oldVault, IDelayVaultProvider _newVault) {
-        require(address(_oldVault) != address(0), "LightMigrator: Invalid old delay vault contract");
-        require(address(_nft) != address(0), "LightMigrator: Invalid lock deal nft contract");
-        require(address(_newVault) != address(0), "LightMigrator: Invalid new delay vault contract");
+    constructor(
+        ILockDealNFT _nft,
+        IDelayVaultV1 _oldVault,
+        IDelayVaultProvider _newVault,
+        string memory _name,
+        string memory _version
```

Comment:
We need to pass it static, and update when version is updated

Reply:
LightMigrator@v0.9.6-ironblocks

or we can do LightMigrator@v1.0.0
to start from fresh deploy